src/screens/Favorites/styles.ts

```diff
@@ -0,0 +1,16 @@
+import {Image} from 'react-native';
+import styled from 'styled-components/native';
+
+export const Container = styled.View`
+  flex: 1;
+  align-items: center;
+  padding: 0 10px;
+`;
+export const BackgroundImage = styled(Image)`
```

Comment:
```suggestion
export const BackgroundImage = styled.Image`
```

THERE IS NO NEED TO IMPORT THE COMPONENT FROM THE REACT NATIVE IF YOU CAN JUST USE IT LIKE STYLED.COMPONENT